### PR TITLE
creates a KMI docker image

### PR DESCRIPTION
**Pull request by jweede**
Creates a functional KMI docker image. Let me know if this is useful.

### CODE REVIEW

**Comment by icemac** [comment]
Anyone who can review this PR?

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+.PHONY: docker-publish  # publishes the docker image
+docker-publish: docker
+	docker push "$(DOCKER_TAG)"
+
+.PHONY: docker-run  # quick example of running this
+docker-run: docker keys/
+	bash docker/run_dev.sh
+
+bin/kmi: bin/buildout
+	$(VENV)/bin/pip install 'setuptools<38.2'
```
Comment (mgedmin) [inline Makefile]: Why limit to an old setuptools version?

**Review by mgedmin — APPROVED** [review]
Eh, lgtm

File: Makefile (hunk 1 above)
Comment (jweede) [reply Makefile]: In the past, setuptools upgrades have broken buildout, so I was probably being defensive. Might be more sensible to limit to `setuptools<39`

**Pull request merged**